### DISCUSSION TITLE
Add button on homepage hero section with link to wallets

### DISCUSSION THREAD

**Discussion by RayTracedObserver** [cardano-foundation/cardano-org#50] [Ideas]
Wallets are how users interact with Dapps on the blockchain. Therefore I believe that it is very important that users do not have to go through a menu in order to find the wallet page.

I suggest replacing the rounded « discover cardano » button from the first page with a « download a wallet » button.

Removing the current « discover cardano » rounded button won’t prevent people from discovering cardano. I think the proper way to discover cardano should be through scrolling down on the first page and not through being redirected to another page. Now that the website is open sourced, the first page will progressively be improved and will more and more accurately represent the spirit, philosophy and tech of cardano. That’s how people will discover cardano. There should be no need to redirect people elsewhere to "discover cardano".

Anyway, wallets need to be more obviously accessible. And the perfect way to do that is to replace the "discover cardano" button with a « download wallet » button which redirects to the wallet page.

**Comment by katomm** [comment]
Thank you for starting this conversation. I'm keen to understand what's your reasoning of featuring Lace.

**Comment by RayTracedObserver** [comment]
> Thank you for starting this conversation. I'm keen to understand what's your reasoning of featuring Lace.

I think discussing that will derail the main subject of the discussion. I will delete that part in the original post and perhaps open another discussion later.

**Reply by katomm** [reply]
Okay. This discussion seems related: #51

**Comment by RayTracedObserver** [comment]
![Screenshot 2024-08-03 at 18 04 51](https://github.com/user-attachments/assets/63598b96-3b70-4fbf-8354-a974be37d33c)
Perhaps adding a third button would be an even better improvement. Rather than replacing discover Cardano as I first had suggested.

There is absolutely enough space on all devices for three buttons and this will be so much more welcoming to new users.

Button could be longer than other buttons for desktop and tablet sized screens so that users who aren't tech savvy and have eye sight issues like my mother for example, can instantly find the get a wallet button. Coding wizards will instantly notice the "start building" button anyway. It looks slightly better with the long button in my opinion than having three equally sized buttons. However I see no opposition at all to having equally sized buttons if that's what others want.

On mobile sized displays the three buttons will always have the same size.

**Comment by katomm** [comment]
I have to admit, it doesn't look particularly appealing to me. Since "Discover Cardano" is already very prominent in the "Learn" menu and other places, you could replace the "Discover Cardano" button with either "What is ada?" (https://cardano.org/what-is-ada) or the completely new "Where to get ada?" page (https://cardano.org/where-to-get-ada/).

**Reply by RayTracedObserver** [reply]
Ok, I've made a pull request with the button change proposal. Keeping two buttons.

**Reply by katomm** [reply]
Just saw it, I think #88  is a good improvement and we will build out the where to get ada page further soon.
